app/models/author.rb

```diff
@@ -0,0 +1,9 @@
+class Author < ActiveRecord::Base
+  def last_5_books
```

Comment:
leave name for now, and change this method after rebasing with master. There will be `Book` model.